code/modules/projectiles/guns/smgs.dm

```diff
@@ -270,12 +270,12 @@
 
 	attachable_offset = list("muzzle_x" = 33, "muzzle_y" = 18,"rail_x" = 12, "rail_y" = 21, "under_x" = 28, "under_y" = 17, "stock_x" = 28, "stock_y" = 17)
 
-	fire_delay = 0.3 SECONDS
+	fire_delay = 0.2 SECONDS
```

Comment:
this makes burst mode redundant (actually worse than full auto).
P.S. this is literally the worst sprite in game, please someone replace it.